jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -215,3 +213,21 @@
                     pattern: '_artifacts/**.xml'
                     deleteoutput: false
         - gcs-uploader
+
+- project:
+    name: kubernetes-pull
+    suffix:
+      - test-unit-integration:
```

Comment:
See, for example, 
https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L57